vmm/src/api/mod.rs

```diff
@@ -195,6 +195,7 @@ pub struct VmReceiveMigrationData {
 pub struct VmSendMigrationData {
     /// URL to migrate the VM to
     pub destination_url: String,
+    pub local: bool,
```

Comment:
Please don't forget to add it to our OpenAPI spec file.

Reply:
Add a comment to explain this attribute's purpose.